README.md

```diff
@@ -8,6 +8,14 @@ MDCT-MCR is an application meant to collect state report data to help the health
 
 Managed Care is a health care delivery system organized to manage cost, utilization, and quality. Medicaid managed care provides for the delivery of Medicaid health benefits and additional services through contractual arrangements between state Medicaid agencies and Managed Care Organizations (MCOs) that accept a set per member per month (capitation) payment for these services.
 
+There are three reports that users complete:
+
+**MCPAR** - (Managed Care Program Annual Report) report for states to complete annually for each Medicaid managed care program
+
+**MLR** - (Medical Loss Ratio) report for states to complete annually or when contract terms change
+
+**NAAAR** - 
```

Comment:
NAAAR - (Network Adequacy and Access Assurances Report) report for states to complete annually, when contract terms change, or there is a significant change in operations